cli/script/command-executor.ts

```diff
@@ -730,33 +926,41 @@ function printDeploymentHistory(command: cli.IDeploymentHistoryCommand, packageH
                 delete packageObject.metrics.totalActive;
             }
         });
-        
+
         printJson(packageHistory);
     } else if (command.format === "table") {
-        printTable(["Label", "Release Time", "App Version", "Mandatory", "Description", "Install Metrics"], (dataSource: any[]) => {
+        var headers = ["Label", "Release Time", "App Version", "Mandatory"];
+        if (command.displayReleasedBy) {
+            headers.push("Released By");
+        }
+
+        headers.push("Description", "Install Metrics");
+
+        printTable(headers, (dataSource: any[]) => {
             packageHistory.forEach((packageObject: Package) => {
                 var releaseTime: string = formatDate(packageObject.uploadTime);
+                var releasedBy: string = packageObject.releasedBy ? packageObject.releasedBy : "";
                 var releaseSource: string;
                 if (packageObject.releaseMethod === "Promote") {
-                    releaseSource = `Promoted ${ packageObject.originalLabel } from "${ packageObject.originalDeployment }"`;
+                    releaseSource = `Promoted ${packageObject.originalLabel} from "${packageObject.originalDeployment}"`;
                 } else if (packageObject.releaseMethod === "Rollback") {
                     var labelNumber: number = parseInt(packageObject.label.substring(1));
                     var lastLabel: string = "v" + (labelNumber - 1);
-                    releaseSource = `Rolled back ${ lastLabel } to ${ packageObject.originalLabel }`;
+                    releaseSource = `Rolled back ${lastLabel} to ${packageObject.originalLabel}`;
                 }
 
                 if (releaseSource) {
                     releaseTime += "\n" + chalk.magenta(`(${releaseSource})`).toString();
                 }
 
-                dataSource.push([
-                    packageObject.label,
-                    releaseTime,
-                    packageObject.appVersion,
-                    packageObject.isMandatory ? "Yes" : "No",
-                    packageObject.description ? wordwrap(30)(packageObject.description) : "",
-                    getPackageMetricsString(packageObject)
-                ]);
+                var row = [packageObject.label, releaseTime, packageObject.appVersion, packageObject.isMandatory ? "Yes" : "No"];
+                if (command.displayReleasedBy) {
+                    row.push(releasedBy);
```

Comment:
What do you think about displaying "You" in this column if the release author was the current user? We could colorize it another color, that way it would be easier to scan for releases that were clearly released by other users. I feel like this would be similar to the way we display "friendly" relative times for the release time column to improve readability.